### ISSUE TITLE
Transition bug on iPhoneX

### ISSUE THREAD

**Issue by AlaeddineOuerteni** [handsomecode/InteractiveSideMenu#82]
When you try to close the sideMenu by swiping it from right to left, the size of the screenshot changes oddly and at the end of the transition the space at the top that is between the statusBar and the toolBar is gone.

Thank you :)

**Comment by masroorelahi** [comment]
Getting the same issue

**Comment by AshleyDeng** [comment]
It's fixed in version 2.3

**Comment by priyonto** [comment]
I am still facing that issue. any help please?

**Comment by ASH19992** [comment]
YES me too , I am facing this issue not only with iPhone X even the rest! 
Any solution ?

**Comment by jyotisirswa** [comment]
Transition issue is still there.
